src/explore-education-statistics-admin/src/components/editable/Comments.tsx

```diff
@@ -229,7 +236,9 @@ const Comments = ({
                       content: Yup.string().required('Enter a comment'),
                     })}
                     onSubmit={async values => {
+                      toggleSubmitting(true);
                       await updateComment(comment.id, values.content);
+                      toggleSubmitting(false);
```

Comment:
Putting the toggles here seems redundant as we're doing this within the method itself

Reply:
ah, yeah of course. Didn't spot that